Source/Core/buildModuleUrl.js

```diff
@@ -5,6 +5,7 @@ import Resource from "./Resource.js";
 
 /*global CESIUM_BASE_URL,define,require*/
 
+// TODO: all these should have JSDoc tags, or mark as private
```

Comment:
I think this TODO was intended to be attached to the two property-functions (?) attached to the exported `buildModuleUrl`.  Basically, you can `import buildModuleUrl from "./buildModuleUrl.js"`, then call `buildModuleUrl.setBaseUrl(...)` or `buildModuleUrl.getCesiumBaseUrl()`, but the JSDoc for two functions is not recognized and doesn't show up in either the generated "globals" documentation, nor in the TS definitions.

My personal recommendation would to refactor the module to stop using `export default` and produce three named exports instead, but that's a significant breaking change that the team would have to discuss.

Reply:
I'd appreciate if you can open a new issue for this. There's already some discussion around whether this function is intended for public use.

Reply:
Hrm.  I found #8922 and #9783 so it looks like your team was already looking at / working on this.  Maybe you should re-open one or both of those until the discussion is finished?

Reply:
Re-opened https://github.com/CesiumGS/cesium/issues/8922.